Retrieve 360 images

BBrunekreeft opened the pull request:
Support for retrieval of 360 images

BBrunekreeft committed:
Update Result.php

Support for retrieval of 360 images

BBrunekreeft committed:
removed double dollar sign

haringsrob commented:
Hallo,

Aangezien 360 iets is dat je afzonderlijk wel wil gebruiken is het misschien beter om hier een aparte method voor te maken?

`has360images`
`get360imageArray`

BBrunekreeft commented:
Lijkt me prima. Ik pas het pull request aan.

BBrunekreeft commented:
Is het dan misschien wat om de videos en manuals dan ook apart op te vragen?
```
hasVideos
getVideos
hasManuals
getManuals
```

BBrunekreeft committed:
Separate functions for retrieval of multimedia objects

Multimedia objects are retrievable with separate functions.
getManuals, getVideos and get360imageArray

BBrunekreeft committed:
Cleaned up whitespace mess

BBrunekreeft committed:
Update readme.md

BBrunekreeft committed:
getMultimediaObjects is public

getMultimediaObjects is public and parameter $objectType is optional, so future objecttypes can be retrieved.
